hw5/task2.js

```
@@ -0,0 +1,6 @@
+const isAdult = (age) => {
+    return age && age >= 18;
```

return age >= 18;

---

you function should return boolean

But I am wondering what did you want to achieve? It might make sense but in particular situations

---

This is just for handling 'undefined' value of age. Looks like it is out of scope so I will leave 'return age >= 18'

---

if it would be undefined then function would return undefined too in previous implementation. I see your point. But we can validate value and return false or throw an error if it is undefined